compliance/azure/azure_long_stopped_instances/README.md

```diff
@@ -16,6 +16,12 @@ The policy leverages the Azure API to check all instances that have been stopped
 ### Required RightScale Roles
 
 - policy_manager
+- credential_viewer
```

Comment:
for consistency please change to Cloud Management - credential_viewer

Reply:
Changed to Cloud Management - credential_viewer